share/completions/yadm.fish

```diff
@@ -0,0 +1,131 @@
+# Completions for yadm v2
+
+# Print an optspec for argparse to handle yadm's options
+# that are independent of any subcommand. Borrowed from git completion file.
+function __fish_yadm_global_optspecs
+    string join \n v-version h/help C= c=+ 'e-exec-path=?' H-html-path M-man-path I-info-path p/paginate \
+        P/no-pager o-no-replace-objects b-bare G-git-dir= W-work-tree= N-namespace= S-super-prefix= \
+        l-literal-pathspecs g-glob-pathspecs O-noglob-pathspecs i-icase-pathspecs \
+        Y/yadm-dir= R-yadm-repo= n-yadm-config= E-yadm-encrypt= a-yadm-archive= B-yadm-bootstrap=
+end
+
+# Config values accepted by `yadm config`
+# See `yadm gitconfig` for setting git config values
+function __fish_yadm_config_keys
+    yadm introspect configs
+end
```

Comment:
I guess this function could now be inlined, whatever you prefer.

Reply:
I think I'll leave it as is, because `__fish_yadm_config_keys` seems more readable and intuitive to me than `yadm introspect configs`